Helper/Config.php

```diff
@@ -292,4 +292,29 @@ public function initSdkConfiguration(bool $isAuthInfoRequired = true) : void
         SdkConfigManager::setValue(sprintf('%s.authorization.auth_user', $sdkEnv), $this->getAuthenticatedUserName());
         SdkConfigManager::setValue(sprintf('%s.authorization.access_token', $sdkEnv), (string) $this->getAccessToken());
     }
+
+    /**
+     * Retrieve a list of stores that the user has access to
+     *
+     * @return $stores array a list of stores
+     */
+    public function getStores()
+    {
+
+        $om = $this->context->getObjectManager();
```

Comment:
I _think_ (very easily could be wrong though) that you can get `StoreManagerInterface` through DI in the constructor, without having to manually get an `ObjectManager` instance

Reply:
Just to chime in with @aldahick, [Magento2's Usage Rules](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/object-manager.html) discourage calling the Object Manager directly.

Reply:
This is fixed